pkgs/applications/science/physics/xnec2c/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+, pkg-config
+, which
+, gtk3
+, openblas
+}:
+
```

```suggestion

assert (!blas.isILP64) && (!lapack.isILP64);
```
ensure we use the LP64 interface.